### PR TITLE
Update syntax demos

### LINKED ISSUE #8: Add missing symbols
Check out the asciimath source code and add missing symbols

### LINKED ISSUE #11: Fix order of greek letters

### LINKED ISSUE #12: Adding examples of underset, overset, stackrel, ubrace, and obrace on the ASCIIMath website
Could you please add usage examples of these on your ASCIIMath website? Thanks..Saf

### LINKED ISSUE #16: Update Grammar
It seems to me as though the grammar for AsciiMath differs from that offered on the site. The implementation offers the following comment.

```
v ::= [A-Za-z] | greek letters | numbers | other constant symbols
u ::= sqrt | text | bb | other unary symbols for font commands
b ::= frac | root | stackrel         binary symbols
l ::= ( | [ | { | (: | {:            left brackets
r ::= ) | ] | } | :) | :}            right brackets
S ::= v | lEr | uS | bSS             Simple expression
I ::= S_S | S^S | S_S^S | S          Intermediate expression
E ::= IE | I/I                       Expression
```

This seems to be a more accurate description of the actual parser behavior. In the grammar defined at the moment `sqrt(x^2+1)` would be invalid since `x^2` is not a simple expression and could not match the `SE` pattern in `E`.

Also why isn't `S/S` defined in the intermediate expression and then let `E ::= IE | I`? Isn't this more correct?

### LINKED ISSUE #19: derivatives using asciimath?
```
d/dx x^2 = 2
```
and 
```
f'(x) = 2x
```
seem to work in the demo.  I wonder if it would be worth adding them to the documentation.

### PR DESCRIPTION
**Pull request by drlippman**
This is a pretty big change, attempting to address several of the lingering issues.  Ideally, this fixes #8, fixes #11, fixes #12, fixes #16, and fixes #19.

I also added columns for the TeX entry alternatives to make it clear those are available, and added additional new symbols that weren't previously captured.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -825,31 +1056,44 @@ <h2>Syntax</h2>
 </table>
 <br/>
 
+<h3>Standard Functions</h3>
+
+<p>Standard functions: sin, cos, tan, sec, csc, cot,
```
Comment (adius) [inline index.html]: You duplicated "standard functions"

**Comment by drlippman** [comment]
Good catch, thanks :)

File: index.html (hunk 1 above)
Comment (drlippman) [reply index.html]: Good catch, thanks :)

**Comment by medovina** [comment]
It's great to see this land - thanks a lot!

**Pull request merged**